rest/kadai-rest-spring/src/main/java/io/kadai/common/rest/ldap/LdapClient.java

```diff
@@ -799,20 +833,20 @@ String[] getLookUpUserInfoAttributesToReturn() {
           getUserOrgLevel4Attribute()
       };
     }
-    return new String[] {
-      getUserIdAttribute(),
-      getUserMemberOfGroupAttribute(),
-      getPermissionNameAttribute(),
-      getUserFirstnameAttribute(),
-      getUserLastnameAttribute(),
-      getUserFullnameAttribute(),
-      getUserPhoneAttribute(),
-      getUserMobilePhoneAttribute(),
-      getUserEmailAttribute(),
-      getUserOrgLevel1Attribute(),
-      getUserOrgLevel2Attribute(),
-      getUserOrgLevel3Attribute(),
-      getUserOrgLevel4Attribute()
+    return new String[]{
+        getUserIdAttribute(),
+        getUserMemberOfGroupAttribute(),
+        getUserPermissionsAttribute(),
```

Comment:
what is this for? Didn't we replace it with permission name?

Reply:
These are the attributes of the user, not the permission. That's why it should be UserPermissionAttribute. Similarly, UserMemeberOfGroupAttribute is used, not the GroupNameAttribute.